app/containers/wallet/WalletSummaryPage.js

```diff
@@ -76,15 +80,40 @@ export default class WalletSummaryPage extends Component<Props> {
       }
     }
 
+    const notification = this._getThisPageActiveNotification();
+
     return (
       <VerticalFlexContainer>
+
+        <NotificationMessage
+          icon={successIcon}
+          show={!!notification.id}
+        >
+          {notification.id && <FormattedHTMLMessage {...notification.message} />}
+        </NotificationMessage>
+
         <WalletSummary
           numberOfTransactions={totalAvailable}
           pendingAmount={unconfirmedAmount}
           isLoadingTransactions={recentTransactionsRequest.isExecutingFirstTime}
         />
+
         {walletTransactions}
+
       </VerticalFlexContainer>
     );
   }
+
+  _getThisPageActiveNotification = (): Notification | any => {
+    let notification = {};
+
+    const { firstActiveNotification } = this.props.stores.uiNotifications;
+    if (firstActiveNotification
```

Comment:
Why do we only care about the first active notification?

Reply:
yeah, maybe i should change it to the most recent notification.

Reply:
Why can you not just check if it contains the notification? What is special about index 0?

Reply:
View component is designed to support only notification at a time